pkg/rtc/subscribedtrack.go

```diff
@@ -17,27 +17,27 @@ const (
 	subscriptionDebounceInterval = 100 * time.Millisecond
 )
 
+type SubscribedTrackParams struct {
+	PublisherIdentity string
+	SubscriberID      string
+	MediaTrack        types.MediaTrack
+	DownTrack         *sfu.DownTrack
+}
 type SubscribedTrack struct {
-	publishedTrack    types.MediaTrack
-	dt                *sfu.DownTrack
-	publisherIdentity string
-	subscriberID      string
-	subMuted          utils.AtomicFlag
-	pubMuted          utils.AtomicFlag
-	settings          atomic.Value // *livekit.UpdateTrackSettings
+	params   SubscribedTrackParams
+	subMuted utils.AtomicFlag
+	pubMuted utils.AtomicFlag
+	settings atomic.Value // *livekit.UpdateTrackSettings
 
 	onBind func()
 
 	debouncer func(func())
 }
 
-func NewSubscribedTrack(mediaTrack types.MediaTrack, publisherIdentity string, subscriberID string, dt *sfu.DownTrack) *SubscribedTrack {
```

Comment:
Ouch ouch ouch, sorry. Thought I had changed this. How did I miss this? :-(

Thank you for taking care of this.

Reply:
no worries! :)